.github/workflows/kubernetes-cd.yml

```diff
@@ -0,0 +1,111 @@
+name: Kubernetes cd
+on:
+  workflow_dispatch:
+    inputs:
+      app_name:
+        type: string
+        required: true 
+        description: Specifies the name of the application to be deployed which is required to reference the correct helm chart and create the job
+      image_tag:
+        type: string
+        required: true
+        description: Specifies the docker image tag which is required to reference the latest image version
+      # Might be needed in the future, if so pre-create namespace but packages are intended for open source  
+      # image_pull_secret_name:
+      #   type: string
+      #   required: true
+      #   description: Specifies the imagePullSecretName which is required for a container registry pull   
+      resource_group_name: 
+        type: string
+        required: true
+        description: Specifies the name of the resource group which is required to authenticate to the AKS cluster
+      kubernetes_cluster_name: 
+        type: string
+        required: true
+        description: Specifies the name of the AKS cluster which is required to authenticate to the AKS cluster
+env:
+  helm_chart_directory_path: ./helm-charts
+  temporary_upload_artifact_directory: ./temp/upload/packages
+  temporary_download_artifact_directory: ./temp/download/packages
+  container_registry_domain: ghcr.io
+  organization_domain_name: buerokratt
+  namespace: applications
+
+jobs:
```

Comment:
There are a few block under here that don't have 2-space indentations. Could you please check that for all lines below here?